pages/home/index.js

```diff
@@ -68,6 +68,7 @@ export default {
         <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/log">Write to the logs</a></li>
         <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/environment">Use environment variables</a></li>
         <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/uncaught-exceptions">Read logs for uncaught exceptions</a></li>
+        <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/context-site">Access Site Information</a></li>
```

Comment:
In keeping with the rest of the conventions, it's probably best to only uppercase "Access" here

Reply:
```suggestion
        <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/context-site">Access site information</a></li>
```